test/unit/full_text_search/issue_query_any_searchable_test.rb

```diff
@@ -0,0 +1,41 @@
+require File.expand_path("../../../test_helper", __FILE__)
+
+module FullTextSearch
+  class IssueQueryAnySearchableTest < ActiveSupport::TestCase
+    setup do
+      unless IssueQuery.method_defined?(:sql_for_any_searchable_field)
+        skip("Required feature 'sql_for_any_searchable_field' does not exist.")
+      end
+    end
+
+    def test_or_one_word
+      Issue.destroy_all
+      issue_with_searched_word_in_subject = Issue.generate!(subject: "Groonga")
```

Comment:
Can we simplify variable name for easy to read? `subject_groonga`, `description_groonga`, `journal_groonga`?